client/types.go

```diff
@@ -59,6 +59,57 @@ const (
 	ConnectConsistencyLevelAll
 )
 
+// String returns the consistency level as a string
+func (l ConnectConsistencyLevel) String() string {
+	switch l {
+	case ConnectConsistencyLevelAny:
+		return "ConnectConsistencyLevelAny"
+	case ConnectConsistencyLevelNone:
+		return "ConnectConsistencyLevelNone"
+	case ConnectConsistencyLevelOne:
+		return "ConnectConsistencyLevelOne"
+	case ConnectConsistencyLevelMajority:
+		return "ConnectConsistencyLevelMajority"
+	case ConnectConsistencyLevelAll:
+		return "ConnectConsistencyLevelAll"
+	}
+	return ""
+}
+
+// ReadConsistencyLevel is the consistency level for reading from a cluster
+type ReadConsistencyLevel int
+
+const (
+	// ReadConsistencyLevelOne corresponds to reading from a single node
+	ReadConsistencyLevelOne ReadConsistencyLevel = iota
+
+	// ReadConsistencyLevelUnstrictMajority corresponds to reading from the majority of nodes
+	// but relaxing the constraint when it cannot be met, falling back to returning success when
+	// reading from at least a single node after attempting reading from the majority of nodes
+	ReadConsistencyLevelUnstrictMajority
+
+	// ReadConsistencyLevelMajority corresponds to reading from the majority of nodes
+	ReadConsistencyLevelMajority
+
+	// ReadConsistencyLevelAll corresponds to reading from all of the nodes
+	ReadConsistencyLevelAll
+)
+
+// String returns the consistency level as a string
+func (l ReadConsistencyLevel) String() string {
+	switch l {
+	case ReadConsistencyLevelOne:
+		return "ReadConsistencyLevelOne"
+	case ReadConsistencyLevelUnstrictMajority:
+		return "ReadConsistencyLevelUnstrictMajority"
+	case ReadConsistencyLevelMajority:
+		return "ReadConsistencyLevelMajority"
+	case ReadConsistencyLevelAll:
+		return "ReadConsistencyLevelAll"
+	}
+	return ""
```

Comment:
Might be better to return "ReadConsistencyLevelUnknown" so it's easier to debug?